### PR TITLE
Update set-up-multi-factor-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Added reference link for how to change 2fa settings

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1694

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The following nine administrator roles will be required to perform additional au
 - User administrator
 - Authentication administrator
 
-All other users will be asked to perform additional authentication when needed.
+All other users will be asked to perform additional authentication when needed. For more information, please visit [Two-factor verification method and settings](https://docs.microsoft.com/azure/active-directory/user-help/multi-factor-authentication-end-user-manage-settings#turn-on-two-factor-verification-prompts-on-a-trusted-device)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
All other users will be asked to perform additional authentication when needed. For more information, please visit [Two-factor verification method and settings](https://docs.microsoft.com/azure/active-directory/user-help/multi-factor-authentication-end-user-manage-settings#turn-on-two-factor-verification-prompts-on-a-trusted-device).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@msbemba  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. This is based on issue #1694.

Thank you.

cc: @kenwith

**Pull request merged**